src/commands/rover/WhoisCommand.js

```diff
@@ -31,11 +31,16 @@ class WhoisCommand extends Command {
   // This is probably the worst file in the entire project, so prepare yourself.
   // TODO: DRY this up and make the method that gets user data a method in DiscordMember
   async fn (msg, args) {
-    const member = args.member
+    let member = args.member
+    // If no member is specified the member is the message author
+    if (!member) member = msg.member
```

Comment:
Why is this behavior wanted like who wants this i dont

Reply:
We (briefly) talked about this in the discord server, do you still want this removed? Or should the user be prompted for a member if they don't supply one? I don't believe no response at all is a good idea.

Reply:
M